discovery-provider/src/queries/get_tracks.py

```diff
@@ -91,8 +100,7 @@ def get_unpopulated_track():
 
             return (tracks, track_ids)
 
-        key = make_cache_key(args)
-        (tracks, track_ids) = use_redis_cache(key, UNPOPULATED_TRACK_CACHE_DURATION_SEC, get_unpopulated_track)
+        (tracks, track_ids) = get_tracks_and_ids()
```

Comment:
should we be adding to cache in the event that we do get stuff from the db here?

Reply:
discussed on slack